src/audio/dai-zephyr.c

```diff
@@ -222,7 +222,7 @@ static int dai_get_fifo(struct dai *dai, int direction, int stream_id)
 }
 
 /* this is called by DMA driver every time descriptor has completed */
-static enum dma_cb_status
+static inline enum dma_cb_status
```

Comment:
same here - this shouldn't matter

Reply:
since only one time calling, add inline at least not wrong, right? :)

Reply:
no, as I said, every change should be justified, "it shouldn't hurt" doesn't look like a valid jastification to me

Reply:
please check previous comments, compiler itself can't do inline operation, add line can reduce the entry/exit function cost.

Reply:
this will not be needed after #8033 merge